internal/status.go

```diff
@@ -0,0 +1,131 @@
+package internal
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type Status_Line struct {
```

Comment:
Golang generally uses CamelCase for variable and class names. Please follow this convention. This means naming this `StatusLine`.